dataset_config/1000_genomes/bigquery.json

```diff
@@ -1,15 +1,53 @@
+// sample_columns:
+//   Columns in sample tables which contain specific types of genomic files.
+//   The names of these files will create special _has_<file_type> fields in the
+//   indexed which is utilized in the 'Samples Overview' facet.
 // table_names:
-//   Fully-qualified table name: <project id>.<dataset id>.<table name>
+//   Fully-qualified BigQuery table names:
+//   Follows the format<project id>.<dataset id>.<table name> (note '.' instead
+//   of ':' between project and dataset).
 {
-  "primary_key": "Sample",
+  "participant_id_column": "Sample",
+  "sample_id_column": "sample_id",
+
+  "sample_file_columns": {
+    "WGS BAM": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.wgs_high_cov_bam",
+    "Mapped BAM": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.mapped_bam",
+    "Mapped CRAM": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.mapped_cram",
+    "Unmapped BAM": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.unmapped_bam",
+    "Chr1 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr1_vcf",
+    "Chr2 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr2_vcf",
+    "Chr3 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr3_vcf",
+    "Chr4 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr4_vcf",
+    "Chr5 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr5_vcf",
+    "Chr6 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr6_vcf",
+    "Chr7 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr7_vcf",
+    "Chr8 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr8_vcf",
+    "Chr9 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr9_vcf",
+    "Chr10 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr10_vcf",
+    "Chr11 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr11_vcf",
+    "Chr12 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr12_vcf",
+    "Chr13 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr13_vcf",
+    "Chr14 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr14_vcf",
+    "Chr15 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr15_vcf",
+    "Chr16 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr16_vcf",
+    "Chr17 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr17_vcf",
+    "Chr18 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr18_vcf",
+    "Chr19 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr19_vcf",
+    "Chr20 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr20_vcf",
+    "Chr21 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr21_vcf",
+    "Chr22 VCF": "bryancrampton-testing.bryan.1000_genomes_samples_manifest.chr22_vcf"
+  },
 
   "table_names": [
```

Comment:
Can table_names come before sample_columns, since former is required? Same for comments.